samples/drivers/ps2/boards/mec1501modular_assy6885.overlay

```diff
@@ -13,3 +13,9 @@
 	wakerx-gpios = <MCHP_GPIO_DECODE_155 GPIO_ACTIVE_HIGH>;
 	wakeup-source;
 };
+
+/ {
+	aliases {
+		ps2-port-0 = &ps2_0;
```

Comment:
hey, looks like most aliases have no dash before the number for some reasons, could you change this to `ps2-port0`?

Reply:
Done.